clients/src/main/java/org/apache/kafka/common/network/KafkaChannel.java

```diff
@@ -108,6 +119,19 @@ public boolean isMute() {
         return muted;
     }
 
+    public boolean isInMutableState() {
+        //some requests (like disconnect) may not require memory allocation.
+        //do we do not want to just mute channels where otherwise the next request received
+        //could be immediately executed regardless of memory.
```

Comment:
Not very clear on the above comment. Is "do we do not" a typo? Is the comment in the right place?